storage-proofs/src/proof.rs

```diff
@@ -88,4 +89,15 @@ pub trait ProofScheme<'a> {
     fn with_partition(pub_in: Self::PublicInputs, _k: Option<usize>) -> Self::PublicInputs {
         pub_in
     }
+
+    fn satisfies_requirements(
+        _pub_params: &Self::PublicParams,
+        _requirements: &Self::Requirements,
+        _partitions: usize,
+    ) -> bool {
+        true
+    }
 }
+
+#[derive(Default)]
+pub struct NoRequirements();
```

Comment:
You can make this an empty struct: `pub struct NoRequirements`, that way you should be able to pass it as `my_fn(&NoRequirements)` instead of the `my_fn(&NoRequirements())`